### PR TITLE
meteor: init at 1.2.0.1

### PR DESCRIPTION
**Pull request by cstrahan**
This packages Meteor 1.2.0.1.

For reference, see: meteor/meteor#5053

/cc @aloiscochard

### CODE REVIEW

**Comment by maurer** [comment]
This procedure seems to have a few issues:
* Downloads binaries during build, making this not a fixed derivation
* Downloads binaries when run, making the software not the same across runs even were the derivation fixed
* Uses copies of nodejs + mongodb, resulting in duplication of software without separate packages

The last one can be overlooked, but the first problem will break basic principles that make the nix store work (derviations with the same hash will produce different trees when evaluated at different times), and the second will break user reasoning about installed programs.

To fix the this problem, you could use `fetchurl` and friends to grab the components `meteor` is going to use from some fixed url+hash, then neuter out all the autoupdate features. I know this is hard, because I tried to do it and failed, but without this it's going to break repeatable builds.

**Comment by cstrahan** [comment]
Just to be clear, the build itself does *not* download binaries (I build everything in a chroot). However, running `meteor` *might*, depending on your configuration and what packages you've already fetched. This is no different than, say, `node`/`npm` where we purely fetch dependencies and register them in the right places, but that of course doesn't stop one from using it imperatively when neaded (e.g. `nix-shell -p nodejs --comand "npm install"`).

To recap, this PR is akin to adding `nodejs`, before tackling the `buildNodePackage` infrastructure at some later point.

**Comment by maurer** [comment]
@cstrahan Just checked the travis build. It looks like you're `fetchzip`ing a tarball. Switching to `fetchurl` should fix the error.

**Comment by cstrahan** [comment]
Strange... It builds for me. Maybe `nox` has applied my commit on a different base commit?

**Comment by cstrahan** [comment]
Alright, I switched to `fetchurl`. Hopefully that'll make CI a little happier.
